checkstyle-tester/diff.groovy

```diff
@@ -289,6 +291,27 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateListOfProjectsHtml(diffDir, listOfProjects) {
+   def diffToolDir = Paths.get("").toAbsolutePath()
+        .parent
+        .resolve("patch-diff-report-tool")
+        .toFile()
+    def diffToolJarPath = getPathToDiffToolJar(diffToolDir)
+    this.class.classLoader.rootLoader.addURL(new URL("file:$diffToolJarPath"))
+    def textTransform = Class.forName("com.github.checkstyle.site.TextTransform").newInstance()
+    def listOfProjectsHtml = new File("$diffDir/listofprojects.html")
```

Comment:
Why not `projects-to-test-on.html`? To me, that makes more sense that we mirror the file name that we are creating a HTML version of.